internal/supervisor/installation.go

```diff
@@ -206,7 +208,13 @@ func (s *InstallationSupervisor) transitionInstallation(installation *model.Inst
 		return s.updateInstallation(installation, instanceID, logger)
 
 	case model.InstallationStateUpdateInProgress:
-		return s.waitForUpdateComplete(installation, instanceID, logger)
+		return s.waitForUpdateStable(installation, instanceID, logger)
+
+	case model.InstallationStateHibernationRequested:
```

Comment:
What is the opposite state, when a Wakeup is requested?

Reply:
It does the normal update flow. This is to ensure that whatever we have in the object configuration is put into the running state.